src/SprykerEco/Zed/Heidelpay/Business/Payment/Request/AdapterRequestFromQuoteBuilder.php

```diff
@@ -49,6 +49,16 @@ public function buildCreditCardRegistrationRequest(QuoteTransfer $quoteTransfer)
         return $registrationRequestTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\HeidelpayRequestTransfer
+     */
+    public function buildEasyCreditRequest(QuoteTransfer $quoteTransfer): HeidelpayRequestTransfer
+    {
+        return $responseTransfer = $this->buildBaseQuoteHeidelpayRequest($quoteTransfer);
```

Comment:
you don't need `$responseTransfer`  variable here

Reply:
Fixed.